src/Tribe/Tickets.php

```diff
@@ -176,6 +176,113 @@ public function get_ticket( $event_id, $ticket_id ) {
 		}
 
 		/**
+		 * Returns the html for the delete ticket link
+		 *
+		 * @since TBD
+		 *
+		 * @param object $ticket Ticket object
+		 */
+		public function get_ticket_delete_link( $ticket = null ) {
+			if ( empty( $ticket ) ) {
+				return;
+			}
+
+			if ( apply_filters( 'tribe_tickets_current_user_can_delete_ticket', true, $ticket->ID, $ticket->provider_class ) ) {
+				$delete_link = sprintf(
+					'<span><a href="#" attr-provider="%1$s" attr-ticket-id="%2$s" id="ticket_delete_%2$s" class="ticket_delete">%3$s</a></span>',
+					$ticket->provider_class,
+					$ticket->ID,
+					esc_html__( 'Delete Ticket', 'event-tickets' )
+				);
+
+				return $delete_link;
+			}
+
+			return;
+		}
+
+		/**
+		 * Returns the url for the move ticket link
+		 *
+		 * @since TBD
+		 *
+		 * @param int $post_id the id of the parent post/event
+		 * @param object $ticket Ticket object
+		 *
+		 * @return string HTML link
+		 */
+		public function get_ticket_move_url( $post_id, $ticket = null ) {
+			if ( empty( $ticket ) || empty( $post_id ) ) {
+				return;
+			}
+
+			$post_url = get_edit_post_link( $post_id, 'admin' );
+
+			$move_type_url = add_query_arg(
+				array(
+					'dialog'         => Tribe__Tickets__Main::instance()->move_ticket_types()->dialog_name(),
+					'ticket_type_id' => $ticket->ID,
+					'check'          => wp_create_nonce( 'move_tickets' ),
+					'TB_iframe'      => 'true',
+				),
+				$post_url
+			);
+
+			return $move_type_url;
+		}
+
+		/**
+		 * Returns the html for the move ticket link
+		 *
+		 * @since TBD
+		 *
+		 * @param int $post_id the id of the parent post/event
+		 * @param object $ticket Ticket object
+		 *
+		 * @return string HTML link
+		 */
+		public function get_ticket_move_link( $post_id, $ticket = null ) {
+			if ( empty( $ticket ) ) {
+				return;
+			}
+
+			$move_url = $this->get_ticket_move_url( $post_id, $ticket );
+
+			if ( ! empty( $move_url ) ) {
```

Comment:
Invert this if and do `return;` inside to reduce nesting around code we actually care about.